### PR TITLE
#2 Basic requirements

### PR DESCRIPTION
**Pull request by carlosmonteagudoinbibo**
This issue delivers the following pages:
- Index
- Faq
- Installation
- Licensing
- Release notes
- Requirements
- Terminology

Secondarily:
- small fixes and adjustments in maya documentation

### CODE REVIEW

File: docs/licensing.md [hunk 1]
```diff
@@ -1,9 +1,357 @@
...
 
 ## Floating Licensing
 
-Abc
\ No newline at end of file
+This section will explain how to configure, run and set-up the licensing server and AdonisFX for leasing floating licenses when the goal is to not restrict the use of AdonisFX to one single machine. AdonisFX floating licensing system requires the use of a license server in charge of providing, dropping and handling licenses from an active lease "pool". When the amount of requested licenses surpass the amount of licenses purchased for that floating license server no further activations of AdonisFX can be made until a lease is dropped and returned to the lease "pool". If for example 20 licenses had been purchased, then one of them will remain leased until the plug-in is unloaded from the target DCC.
+
+It is possible to run the license server on one operating system and run instances of AdonisFX on a different operating system.
+
+The licensing server is provided and shipped with the installation of AdonisFX for x64 architectures and can be run on Windows, Linux and MacOS. For more builds please visit [https://wyday.com/download/](https://wyday.com/download/).
```
Comment (lucasferryinbibo) [inline docs/licensing.md:210]: This is currently not true. We only shup the .exe for Windows and not for Linux. Do we want to add the other.exe files or just refer to Windows x64 here?

**Comment by lucasferryinbibo** [comment]
We have to review some points in the licensing documentation and FAQ's as they are more managerial questions rather than licensing questions. For example exposing that trial licenses are not for commercial use or if the user can re-request a trial license, for example. Other than that #2 in terms of Licensing page is completed on my end.

File: docs/licensing.md [hunk 2]
```diff
@@ -1,9 +1,364 @@
...
+- **Floating Licensing:** License mode that allows different users to request licenses from a common license pool (served by a server) avoiding so the use restriction of the software to a single machine.
+
+AdonisFX also distinguishes between two *Licensing Types*:
+
+- **Interactive:** License type that allows the user to use AdonisFX from the graphical interface of the software where AdonisFX was loaded. This licensing type is intended for users that want to build scenes, manipulate and interact with AdonisFX using visual and interactive feedback.
+- **Batch:** License type that allows the user to use AdonisFX from a batch script or terminal without the ability to use the graphical interface for manipulating the software. This licensing type is intended for users that want to run AdonisFX from a terminal to for example render a scene on the farm after setting the scene up using an Interactive License.
+
+To be able to activate AdonisFX it is required to purchase a `PRODUCT KEY`. Product keys can be purchased through Inbibo's official website [https://www.inbibo.co.uk/](https://www.inbibo.co.uk/). AdonisFX product keys have the following characteristics:
```
Comment (lucasferryinbibo) [inline docs/licensing.md:16]: What will the purchase address be for the licenses?

File: docs/licensing.md [hunk 3]
```diff
@@ -1,9 +1,364 @@
...
+
+### Trial
+
+AdonisFX allows the user to use the product for **30 days** in Node-Locked Interactive mode. This means that the trial can be used using the graphical interface for one single machine at a time. The trial period requires activation which can be handled in an online or offline way.
+
+Trial licenses are intended for testing and non-commercial purposes. To use AdonisFX for commercial purposes a `PRODUCT KEY` must be purchased through Inbibo's official website [https://www.inbibo.co.uk/](https://www.inbibo.co.uk/) and activated.
+
+To request an extension of the trial license period contact support at **adnsupport@inbibo.co.uk**.
+
+See the **AdonisFX EULA** for details.
```
Comment (lucasferryinbibo) [inline docs/licensing.md:157]: Should we link here the EULA or remove this line?

File: docs/licensing.md [hunk 4]
```diff
@@ -1,9 +1,364 @@
...
+    - The `PRODUCT KEY` has the following format: **XXXX-XXXX-XXXX-XXXX-XXXX-XXXX-XXXX**
+    - Once the activation period has concluded, a new activation request and response have to be generated.
+
+**NOTE:** For deactivating licenses, please contact support.
+
+### Trial
+
+AdonisFX allows the user to use the product for **30 days** in Node-Locked Interactive mode. This means that the trial can be used using the graphical interface for one single machine at a time. The trial period requires activation which can be handled in an online or offline way.
+
+Trial licenses are intended for testing and non-commercial purposes. To use AdonisFX for commercial purposes a `PRODUCT KEY` must be purchased through Inbibo's official website [https://www.inbibo.co.uk/](https://www.inbibo.co.uk/) and activated.
```
Comment (lucasferryinbibo) [inline docs/licensing.md:153]: I've added `non-commercial` but I'm not sure if sentences like this one are aligned with the EULA. let's double check

File: docs/licensing.md (hunk 1 above)
Comment (carlosmonteagudoinbibo) [reply docs/licensing.md]: We are going to include all binaries for windows and linux (today hopefully). I would remove "MacOS" from the sentence. And yes, we will provide the binaries for 64-bit. Other architectures will have to be downloaded from that link

File: docs/faq.md [hunk 5]
```diff
@@ -1,3 +1,99 @@
...
+Yes, you can use the AdonisFX [Export](documentation/maya/tools.md#adonisfx-export-tool) and [Import](documentation/maya/tools.md#adonisfx-import-tool) Tools to generate and load respectively an Adonis Asset Definition (AAD) file. From an scene with an asset fully configured with Adonis deformers, you can select the parent transform object and launch the Exporter. From there, you can decide which settings to export and the destination path for the AAD file. Then, in a different scene with a compatible asset, you can launch the Importer, browse the AAD file and assign deformers from the file to the meshes in the hierarchy of the asset.
+
+#### For what units is Adonis designed?
+
+Adonis solvers interpret input units as meters. This means that in order to simulate external forces, the Space Scale may need to be adjusted. For example, to apply Gravity with a value of 9.8 m/s<sup>2</sup>, the Space Scale should be set to 0.01.
+
+#### Can I activate AdonisFX offline?
+Yes. Visit the [licensing](licensing.md#licensing) page for more information.
+
+#### Can I renew my 30 days trial license period?
```
[the remark this answers is not in the document]
Comment (carlosmonteagudoinbibo) [reply docs/faq.md:64]: Let's remove this question for now. It's a "no" at the moment. I will check with Craig/Marco

File: docs/licensing.md (hunk 2 above)
Comment (carlosmonteagudoinbibo) [reply docs/licensing.md:16]: We don't have the final link yet. Add a "PENDING" for now

File: docs/licensing.md (hunk 3 above)
Comment (carlosmonteagudoinbibo) [reply docs/licensing.md:157]: There will be a link in the website too. Add a "PENDING" for now

File: docs/licensing.md (hunk 4 above)
Comment (carlosmonteagudoinbibo) [reply docs/licensing.md:153]: I will double check with Craig. Thanks Lucas

File: docs/faq.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (lucasferryinbibo) [reply docs/faq.md:64]: Done!

File: docs/licensing.md (hunk 1 above)
Comment (lucasferryinbibo) [reply docs/licensing.md]: Done!

File: docs/licensing.md (hunk 2 above)
Comment (lucasferryinbibo) [reply docs/licensing.md:16]: Done!

File: docs/licensing.md (hunk 3 above)
Comment (lucasferryinbibo) [reply docs/licensing.md:157]: Done!

**Review by lucasferryinbibo — APPROVED** [review]

**Pull request merged**